### PR TITLE
BC-8396 - Adjust BoardTile design (2)

### PR DESCRIPTION
**Pull request by odalys-dataport**
# Short Description

## Links to Ticket and related Pull-Requests

BC-8396

## Screenshots of UI changes

Screenshot 2024-12-05 at 12 36 59

## Checklist before merging

- [x] QA: In addition to review, the code has been manually tested (if manual testing is possible)
- [ ] PO: Any deviation from requirements was agreed with Product-Owner / ticket author / support-team
- [ ] DEV: Every new component is implemented having accessibility in mind (e.g. aria-label, role property)

> Notice: Please keep this Pull-Request as a Draft (or add WIP label), until it is ready to be reviewed

### CODE REVIEW

File: src/modules/feature/room/RoomMenu.unit.ts [hunk 1]
```diff
@@ -0,0 +1,169 @@
...
+				plugins: [
+					createTestingVuetify(),
+					createTestingI18n(),
+					createTestingPinia(),
+				],
+				stubs: ["RouterLink"],
+			},
+			attachTo: document.body,
+			stubs: {
+				RouterLink,
```
Comment (MartinSchuhmacher) [inline src/modules/feature/room/RoomMenu.unit.ts:34]: Do we need both stubs?

File: tests/test-utils/factory/room/roomFactory.ts [hunk 2]
```diff
@@ -0,0 +1,14 @@
+import { Factory } from "fishery";
+import { RoomDetails, RoomColorEnum } from "@/types/room/Room";
+
+export const roomFactory = Factory.define<RoomDetails>(({ sequence }) => ({
+	id: `room${sequence}`,
+	name: `room #${sequence}`,
+	color: RoomColorEnum.BlueGrey,
+	schoolId: "6749dd4e657d98af622e370c",
```
Comment (MartinSchuhmacher) [inline tests/test-utils/factory/room/roomFactory.ts:8]: We also use a "random" id in roomDetails I guess. Not a big fan tbh, would prefer something like 'school${sequence}'

File: src/modules/feature/room/RoomMenu.unit.ts (hunk 1 above)
Comment (odalys-dataport) [reply src/modules/feature/room/RoomMenu.unit.ts]: Oh no, we don't of course, thanks!

File: tests/test-utils/factory/room/roomFactory.ts (hunk 2 above)
Comment (odalys-dataport) [reply tests/test-utils/factory/room/roomFactory.ts:8]: Will do!

**Review by MartinSchuhmacher — APPROVED** [review]

**Review by MartinSchuhmacher — APPROVED** [review]

**Pull request merged**